yarn-project/scripts/package.local.json

```diff
@@ -6,4 +6,4 @@
     "generate:why-these-noops": "echo These noops are here because `yarn workspaces foreach` runs the specified command in the packages that contain it only if two or more packages define it, otherwise it's run everywhere. So we just define these noops as commands to ensure they behave as they should when running watch.sh.",
     "generate:why-these-comments": "echo JSON does not support comments, so we just define these commands for documentation sake."
   }
-}
\ No newline at end of file
+}
```

Comment:
<!--__GRAPHITE_HTML_TAG_START__--><i class='graphite__hidden'>[Re: line 10]<br/><br/></i>

<!--__GRAPHITE_HTML_TAG_END__-->🤷 <!--__GRAPHITE_HTML_TAG_START__--><span class='graphite__hidden'><br/><br/>See this comment inline on <a href="https://app.graphite.dev/github/pr/AztecProtocol/aztec-packages/5292?utm_source=unchanged-line-comment">Graphite</a>.</span>

<!--__GRAPHITE_HTML_TAG_END__-->